<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayoutControllerSettingsWindow</class>
 <widget class="QDialog" name="LayoutControllerSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layout Controller Settings</string>
  </property>
  <widget class="QTabWidget" name="mainTabs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>155</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="general">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QCheckBox" name="gAutoStartCheckBoxLs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Decides if the module will start with main program</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Start with Chiya</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="gChangeRegistryCheckBoxLs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>If this function is enabled, default windows switching will be disabled. Can be used only if Chiya is runned as administrator</string>
     </property>
     <property name="text">
      <string>Disable windows default switching</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gApplyButtonLs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gCurrectCombinationGroupBoxLs">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>25</y>
       <width>171</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Current Combination</string>
     </property>
     <widget class="QRadioButton" name="gNotSetButtonLs">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Not Set</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="gCtrlShiftButtonLs">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CTRL + SHIFT</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="gShiftAltButtonLs">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>LALT + SHIFT</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="gWarningLs">
     <property name="geometry">
      <rect>
       <x>297</x>
       <y>5</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>This will affect your System!</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="gEnableQtAppsCheckBoxLs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>211</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>If this function is enabled, Chiya will work in apps that are built using Qt. Some Qt apps may hang when trying to swith layout in them.</string>
     </property>
     <property name="text">
      <string>Enable Qt apps</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutSpecific">
    <attribute name="title">
     <string>Layout specific</string>
    </attribute>
    <widget class="QCheckBox" name="lsActiveCheckBoxLs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>85</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Decides if this layout can be choosed with default language switching</string>
     </property>
     <property name="text">
      <string>Skip</string>
     </property>
    </widget>
    <widget class="QLabel" name="lsShortcutSelectLabelLs">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>165</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Switch to this keyboard layout</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="lsShortcutActivateButtonLs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>165</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lsShortcutSelectLineEditLs">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>35</y>
       <width>165</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Shortcut to direct switch to this layout</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>No shortcut</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lsApplyButtonLs">
     <property name="geometry">
      <rect>
       <x>262</x>
       <y>90</y>
       <width>83</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lsShortcutActivateLineEditLs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>165</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Shortcut for skip/not skip this layout</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>No shortcut</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lsShortcutSelectButtonLs">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>165</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QListView" name="lsLayoutsListViewLs">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>9</y>
       <width>120</width>
       <height>101</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>List of keyboard layouts in your system</string>
     </property>
    </widget>
    <widget class="QLabel" name="lsShortcutActivateLabelLs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>165</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable keboard layout</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="exceptions">
    <attribute name="title">
     <string>Exceptions</string>
    </attribute>
    <widget class="QPlainTextEdit" name="eExceptionsPlainTextEditLs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>200</width>
       <height>110</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>List of exceptions. Exceptions are divided with space. Write here window names and not .exe</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Chiya Chrome Yurr</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="eWhiteListCheckBoxLs">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>If this is checked, Chiya will work only in apps in list</string>
     </property>
     <property name="text">
      <string>Whitelist</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eApplyButtonLs">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>35</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
